src/CLI/actioner/sonic-cli-arp-neigh.py

```diff
@@ -104,28 +107,72 @@ def run(func, args):
                 if neighs is None:
                     return
 
+                for neigh in neighs:
+                    ext_intf_name = "-"
+                    ipAddr = neigh['state']['ip']
+                    if ipAddr is None:
```

Comment:
You would see an exception if the 'ip' field doesnt exist.

Reply:
yes, that's why it's in a 'try' block. Also, as ip is a key for this table, the ip should be there always. If not, then the table is in invalid state :).